.github/workflows/prune-images.yml

```diff
@@ -1,23 +1,23 @@
 name: Prune Images
-run-name: Prune old images from OpenShift
+run-name: Pruning all but the most recent ${{ inputs.minTags }} tags prior to what is deployed in ${{ inputs.environment }}
 
 on:
   workflow_dispatch:
     inputs:
       applications:
-        description: 'Applications to prune'
+        description: 'Comma seperated list of applications to prune'
         required: true
-        default: 'web-sims, api-sims'
+        default: 'web-sims, api-sims, queue-consumers-sims, workers-sims'
```

Comment:
can we have an input called as "all" to prune in a single go? rather selecting each and every time?

Reply:
This is a single input to prune everything but you have to specify the list.  The reason for the list vs ALL is that each environment is unique.